src/pint/pint_matrix.py

```diff
@@ -607,8 +607,7 @@ def combine_design_matrices_by_param(matrix1, matrix2, padding=0.0):
     for d_param in matrix2.derivative_params:
         if d_param in base_params:
             raise ValueError(
-                "Input design matrix {} has duplicated "
-                "parameters with matrix {}".format(ii, 0)
+                f"Unable to combine the two design matrices as they have duplicated parameter(s). The duplicated parameter is {d_param}."
```

Comment:
Could I suggest a slight modification to make it easier to parse:

f"Unable to combine the two design matrices as they have duplicated parameter(s): '{d_param}'"

Reply:
Done.